projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func Execute() {
+	// define a flag of type boolean with the specific properties (name, value, usage)
+	helpFlag := flag.Bool("h", false, "show help")
+
+	// parse the command line flags
+	flag.Parse()
+
+	// if the "-h" flag is provided
+	if *helpFlag {
+		fmt.Fprintf(os.Stdout, "go-ls help message\n")
+		return
+	}
+
+	// get the non-flag command line arguments
+	args := flag.Args()
+
+	// make a slice of paths to iterate over (if more than one argument is passed)
+	var paths []string
+
+	if len(args) == 0 {
+		// if there are no arguments passed in then use the current working directory
+		workingDirectory, _ := os.Getwd()
+		paths = append(paths, workingDirectory)
+	} else {
+		// if there are arguments
+		paths = append(paths, args...)
+	}
+
+	// loop over the paths
+	for index, path := range paths {
+		// get the "FileInfo" about the path
+		pathInfo, err := os.Stat(path)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "could not read the file/directory information: %v\n", err)
+			os.Exit(2)
+		}
+
+		// if the path is a file then print it back to the user
+		if !pathInfo.IsDir() {
+			fmt.Fprintf(os.Stderr, "%v\n", os.Args[1])
+			break
```

Comment:
Why to stderr, and why `break`?

I would expect:
`% ls /dev/null /dev/random`
to print out the names of both files to stdout?